.github/workflows/deploy.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: "14.x"
+          node-version: "16.x"
```

Comment:
can we bump `actions/setup-node` to v4? then you can use `node-version-file` and it will use what is defined there

Reply:
Great idea thanks!